src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -491,6 +491,10 @@ public void forceSyncSubscriptionsForOrgAsync(String orgId) {
 
   @Transactional
   public void forceSyncSubscriptionsForOrg(String orgId, boolean paygOnly) {
+    if (paygOnly && !properties.isEnablePaygSubscriptionForceSync()) {
+      log.debug("Force sync of payg subscriptions disabled");
```

Comment:
Think we should up the logger level for this since by default the debug level doesn't make it to splunk or anything?

Reply:
Yeah that makes sense to me, I'll make it info.